thoth/storages/graph/postgres.py

```diff
@@ -5115,6 +5117,13 @@ def get_pi_count(self, component: str) -> Dict[str, int]:
 
         return result
 
+    def get_entity_count(self, entity: Union[Base, BaseExtension]) -> int:
```

Comment:
I think the docstring is not good, what does this method do?

Reply:
Updated